config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_computebackendservices.compute.cnrm.cloud.google.com.yaml

```diff
@@ -1166,7 +1166,8 @@ spec:
                       - external
                     properties:
                       external:
-                        description: 'Allowed value: The `name` field of a `NetworkSecurityClientTLSPolicy`
+                        description: 'Allowed value: string of the format `//networksecurity.googleapis.com/projects/{{project}}/locations/{{location}}/clientTlsPolicies/{{value}}`,
```

Comment:
Might be good to make this sort of description more of a standard. Seems like a huge improvement. Worth noting that for this resource it seems like location always has to be "global".

Reply:
ClientTlsPolicy does support other locations([API doc](https://cloud.google.com/service-mesh/v1.22/docs/reference/network-security/rest/v1beta1/projects.locations.clientTlsPolicies#ClientTlsPolicy)). 

If a user incorrectly configures a regional policy, the request should be sent to the API as is and let the API to handle the error. We may not want to convert the regional policy to a global policy on our side(?). In rare cases, there might be a global resource with the same name, and this could lead to confusion.